lib/rules/jest/no-if.js

```diff
@@ -0,0 +1,93 @@
+const {docsUrl} = require('../../utilities');
+
+const TEST_FUNCTION_NAMES = [
+  'it',
+  'xit',
+  'fit',
+  'test',
+  'xtest',
+  'describe',
```

Comment:
I am not sure you need to include `describe` blocks

Reply:
I figured I'd leave it cause it's not like we want to allow:
```
describe('SomeThing', () => {
  if(!someFlag) {
     it('testing', () => {
     //...
     })
  }
})

```